HotSpot/Models/TimeInterval.h

```diff
@@ -0,0 +1,20 @@
+//
+//  TimeInterval.h
+//  HotSpot
+//
+//  Created by drealin on 7/19/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface TimeInterval : PFObject
+@property (nonatomic, strong) NSDate *startTime;
+@property (nonatomic, strong) NSDate *endTime;
+@property (nonatomic, strong) NSNumber *repeatsWeekly;
```

Comment:
repeats weekly is a boolean right?
```suggestion
@property (nonatomic, assign, readwrite) BOOL repeatsWeekly;
```